streamwebs_frontend/streamwebs/models.py

```diff
@@ -233,6 +232,83 @@ class Meta:
         verbose_name_plural = 'Water Quality'
 
 
+class CameraPointManager(models.Manager):
+
+    def create_camera_point(self, site, cp_date, created_by='', latitude=None,
+                            longitude=None, map_datum='', description=''):
+
+        return self.create(site=site, cp_date=cp_date, created_by=created_by,
+                           latitude=latitude, longitude=longitude,
+                           map_datum=map_datum, description=description)
+
+
+class CameraPoint(models.Model):
+    site = models.ForeignKey(Site, on_delete=models.CASCADE, null=True)
+    cp_date = models.DateField(default=datetime.date.today)
+    created_by = models.CharField(max_length=255, blank=True)
+    latitude = models.DecimalField(default=0, max_digits=5, decimal_places=2,
+                                   blank=True, null=True)
+    longitude = models.DecimalField(default=0, max_digits=5, decimal_places=2,
```

Comment:
I don't know about the client but I think we could use more decimal places for the lats and lons. Right now this config gives us a XXX.XX number and I think the students could use something around XXX.XXXXXX (9 digits, 6 decimals)

> The first decimal place is worth up to 11.1 km: it can distinguish the position of one large city from a neighboring large city.
> The second decimal place is worth up to 1.1 km: it can separate one village from the next.
> The third decimal place is worth up to 110 m: it can identify a large agricultural field or institutional campus.
> The fourth decimal place is worth up to 11 m: it can identify a parcel of land. It is comparable to the typical accuracy of an uncorrected GPS unit with no interference.
> The fifth decimal place is worth up to 1.1 m: it distinguish trees from each other. Accuracy to this level with commercial GPS units can only be achieved with differential correction.
> The sixth decimal place is worth up to 0.11 m: you can use this for laying out structures in detail, for designing landscapes, building roads. It should be more than good enough for tracking movements of glaciers and rivers. This can be achieved by taking painstaking measures with GPS, such as differentially corrected GPS.